toolchain/sem_ir/name_scope_test.cpp

```diff
@@ -0,0 +1,261 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "toolchain/sem_ir/name_scope.h"
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+namespace Carbon::SemIR {
+namespace {
+
+using testing::ElementsAre;
+using testing::Pair;
```

Comment:
```suggestion
using ::testing::ElementsAre;
using ::testing::Pair;
```

For context, see https://abseil.io/tips/119

Reply:
Done.